app/src/main/assets/html/error/unable-to-connect.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Unable to Connect</title>
+    <body>
+        <script src="js/error.js"></script>
+        <div class="container">
+            <img class="icon" src="images/icon-errorpage-unabletoconnect.svg" alt="Unable to Connect">
+
+            <h1>Unable to Connect</h1>
+
+            <p>Firefox Reality can’t establish a connection to the
+            <br/>server at <b id="errorURL">www.abc.com</b></p>
+
+            <ol>
+                <li>The site could be temporarily unavailable or too busy. Try again in a few moments.</li>
+                <li>If you are unable to load any pages, check your device’s data or Wi-Fi connection.</li>
```

Comment:
in Firefox release, this is what appears:

> • The site could be temporarily unavailable or too busy. Try again in a few moments.
> • If you are unable to load any pages, check your computer’s network connection.
> • If your computer or network is protected by a firewall or proxy, make sure that Firefox is permitted to access the Web.

![image](https://user-images.githubusercontent.com/203725/42784830-3ab82dd0-8905-11e8-9e0a-c87e3b5b1f86.png)

should we match the text here to be the same text that's in Firefox?